src/components/reactExamples/state.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+
+const State = () => {
+  const [count, setCount] = useState(0);
+
+  return (
+    <div
+      style={{
+        height: "100vh",
+        display: "flex",
+        justifyContent: "center",
+        alignItems: "center",
+        flexDirection: "column",
+      }}
+    >
+      <p>{count}</p>
+      <button
+        onClick={() => setCount((prevCount) => prevCount + 1)}
+        disabled={count >= 5 ? true : false}
```

Comment:
no need for ternary operator, `count >= 5` statement already returns a boolean value
```suggestion
        disabled={count >= 5}
```

Reply:
Done